index.js

```diff
@@ -0,0 +1,72 @@
+var keys = document.getElementsByClassName('keys')
+var operators = ['+', '-', 'x', '÷'];
+var decimalAdded = false;
+
+for( var i = 0; i < keys.length; i++ ) {
+  keys[ i ].onclick = function ( e ) {
+    var input = document.querySelector( '.screen' );
+		var inputVal = input.innerHTML;
```

Comment:
i would change this tabbing so all the variable declarations are at the same starting point. 
i also like to say event instead of passing through e on line 6 just so that i am explicit